db/migrations/001_trackers_create.rb

```diff
@@ -0,0 +1,13 @@
+require 'sequel'
+
+Sequel.migration do 	
+  change do
+  	create_table(:trackers) do
+  	  primary_key :id
+  	  foreign_key :campaign_id
+  	  
+  	  String :label, unique: true, null: false
+  	  String :url, unique: true, null: false
+  	end
+  end
+end
```

Comment:
you need to put an empty line at this end of your files there is this issue in all your migrations files